gulpfile.babel.js

```diff
@@ -174,7 +174,7 @@ gulp.task('scripts', () => {
         // Remove default exports
         .pipe(replace(/(^|\n)export default \w+;/g, ''))
         // Replace imports with references
-        .pipe(replace(/(^|\n)import .* from '(.*)((.d)?.ts)?';/g, '$1/// <reference path="$2.d.ts"/>'))
+        .pipe(replace(/(^|\n)import (.* from )?'(.*)((.d)?.ts)?';/g, '$1/// <reference path="$2.d.ts"/>'))
```

Comment:
Should this be using the 3rd match now?
```suggestion
        .pipe(replace(/(^|\n)import (.* from )?'(.*)((.d)?.ts)?';/g, '$1/// <reference path="$3.d.ts"/>'))
```

Reply:
Good catch. Not sure how it didn't crash out on me. Just goes to show how loose the `scripts` code is when it comes to checking type safety I guess